proposals/0014-wave-size-range.md

```diff
@@ -0,0 +1,283 @@
+# Wave Size Range
+
+* Proposal: [0014](0014-wave-size-range.md)
+* Author(s): [Greg Roth](https://github.com/pow2clk)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8, validator version 1.8
+
+## Introduction
+
+Shader Model 6.6 included the ability to specify a wave size on a shader entry
+ point in order to indicate that this shader depends on a specific wave size.
+If that size of wave isn't available to the current hardware,
+ shader will fail to load.
+Existing queries allow the developer to determine if this will happen in
+ advance.
+This enables creating custom shaders optimized for the wave size of a
+ particularly interesting platform and loading them when possible.
+
+## Motivation
```

Comment:
I think we have a couple of example use cases we know about that we could describe in the motivation here, and I think we're missing discussion about the driver having to choose a wave size on hardware that supports that.